main.go

```diff
@@ -310,8 +317,16 @@ func mustLoadRouter(logger *zap.Logger, config *Config, indexer Indexer) *mux.Ro
 }
 
 func getRouter(logger *zap.Logger, config *Config, indexer Indexer) (*mux.Router, error) {
-	artifactsHandler := artifactsHandler(indexer, config.CacheTimeCatchAll)
-	signaturesHandler := signaturesHandler(indexer, config.CacheTimeCatchAll)
+	proxyMode, err := proxymode.NewProxyMode(proxymode.ProxyOptions{
+		Enabled: featureProxyMode,
```

Comment:
As this is on technical preview, should we add a warning here when enabled?

Reply:
Fixed